File: app/src/main/java/com/mirogal/cocktail/presentation/ui/detail/ingredientlist/ItemHolder.java
package com.mirogal.cocktail.presentation.ui.detail.ingredientlist;

import android.annotation.SuppressLint;
import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.mirogal.cocktail.R;
import com.mirogal.cocktail.presentation.ui.detail.ingredientlist.entity.IngredientEntity;

class ItemHolder extends RecyclerView.ViewHolder {

    private final TextView tvPosition;
    private final TextView tvIngredient;
    private final TextView tvMeasure;

    ItemHolder(@NonNull View itemView) {
        super(itemView);
        tvPosition = itemView.findViewById(R.id.tv_position);
        tvIngredient = itemView.findViewById(R.id.tv_ingredient);
        tvMeasure = itemView.findViewById(R.id.tv_measure);
    }

    @SuppressLint("SetTextI18n")
    void bind(IngredientEntity ingredientEntity) {
        tvPosition.setText(ingredientEntity.getId() + ".");
        tvIngredient.setText(ingredientEntity.getName());
        tvMeasure.setText(ingredientEntity.getMeasure());
    }
}

File: app/src/main/java/com/mirogal/cocktail/presentation/ui/detail/ingredientlist/ItemHolder.kt
package com.mirogal.cocktail.presentation.ui.detail.ingredientlist

import android.annotation.SuppressLint
import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.mirogal.cocktail.R
import com.mirogal.cocktail.presentation.ui.detail.ingredientlist.entity.IngredientEntity

class ItemHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    private val tvPosition: TextView = itemView.findViewById(R.id.tv_position)
    private val tvIngredient: TextView = itemView.findViewById(R.id.tv_ingredient)
    private val tvMeasure: TextView = itemView.findViewById(R.id.tv_measure)

    @SuppressLint("SetTextI18n")
    fun bind(ingredientEntity: IngredientEntity) {
        tvPosition.text = ingredientEntity.id.toString() + "."
        tvIngredient.text = ingredientEntity.name
        tvMeasure.text = ingredientEntity.measure
    }

}